### PR TITLE
feat: improve pay per result docs

### PR DESCRIPTION
**Pull request by mhamas**
Adding information about where to find actor-level stats and how to set maximum number of items an Actor should return.

### CODE REVIEW

File: sources/platform/actors/running/store.md [hunk 1]
```diff
@@ -117,6 +117,11 @@ Under the **pay per result** model, all platform costs generated **during the ru
 
 You will still be charged for the timed storage of the data in the same fashion as with any other Actor. You can always decide to delete the dataset to reduce your costs after you export the data from the platform. By default, any unnamed dataset will be automatically removed after your data retention period, so in most cases, this is nothing to worry about.
 
+**Can I set a cap on how many results an Actor should return**
+You can set a limit on how many items an Actor should return and you will be charged for in Options on the Actor detail page, in the section below the input section.
```
Comment (TheoVasilis) [inline sources/platform/actors/running/store.md:121]:
```suggestion
You can set a limit on how many items an Actor should return and the amount you will be charged in Options on the Actor detail page in the section below the Actor input.
```

File: sources/platform/actors/running/store.md [hunk 2]
```diff
@@ -133,7 +138,10 @@ On the top of that, you can see how much you have been charged for a specific ru
 
 ![Run cost shown on the overview of all runs](./images/store/pay_per_result_run_list.png)
 
-<!-- TODO - add info about where to see this at per-actor level -->
+If you wish to see how much you have been charged for a particular actor, you will find this information
```
Comment (TheoVasilis) [inline sources/platform/actors/running/store.md:141]:
```suggestion
If you wish to see how much you have been charged for a particular Actor, you will find this information
```

File: sources/platform/actors/running/store.md [hunk 3]
```diff
@@ -133,7 +138,10 @@ On the top of that, you can see how much you have been charged for a specific ru
...
+on the bottom of the [Usage tab](https://console.apify.com/billing).
```
Comment (TheoVasilis) [inline sources/platform/actors/running/store.md:142]:
```suggestion
at the bottom of the [Usage tab](https://console.apify.com/billing).
```

**Review by TheoVasilis — CHANGES_REQUESTED** [review]

File: sources/platform/actors/running/store.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mhamas) [reply sources/platform/actors/running/store.md]: solved by Theo's change

**Review by TheoVasilis — APPROVED** [review]

**Review by novotnyj — APPROVED** [review]

**Pull request merged**